### ISSUE TITLE
What would a "modern science notebook" look like?

### ISSUE THREAD

**Issue by ergonyc** [cjsifuen/self-guided-learning#3]
What would be the best practices for doing modern reproducable (and scalable?) science?

- Jupyter notebooks?
- github/gitlab repos?
- published FAIR datasets?
- ???

issues:

- on cloud / off cloud
- change management journey.  how to progress from newbie to ninja?

**Comment by cjsifuen** [comment]
This is a good idea.

I was going to add in a domain-specific series on "reproducibility" which would cover some of this.

If we consider a notebook as part of a reproducible approach to science, we should also add in workflow managers (nextflow, snakemake), environment management (conda and docker), etc.

**Comment by ergonyc** [comment]
I haven't found any "science thought leaders" really pushing anything like this... but my personal opinion would be a jupyter notebook + github setup for ALL scientists to leverage.  However, creating such guidelines is probably out of the scope of the current goal of this repo.  It might really also be outside the mandate of our DSC program.  But I wanted to lodge the idea here.

**Comment by ergonyc** [comment]
For now it might be prudent to simply have a short page of "suggested tools for everyone":
- jupyter notebooks
- github / git
- etc.

**Comment by cjsifuen** [comment]
Agreed. I think it's important to push/add. For the initial timeline, it would be best to stick to things that are easy to reference.
